client/reader/site-stream/featured.jsx

```diff
@@ -24,8 +26,37 @@ export default React.createClass( {
 
 	getStateFromStores( store ) {
 		store = store || this.props.store;
+
+		let postKeys = store.get();
+		let posts = postKeys.map( postKey => {
```

Comment:
since it doesn't look like we end up using `postKeys` or reassigning it or `posts`, we could remove some work here and use `const`

```js
const posts = store.get().map( postKey => {
	…
} );
```